data/brands/amenity/bank.json

```
@@ -8078,6 +8078,18 @@
         "short_name": "PenFed"
       }
     },
+    {
+      "displayName": "Pennsylvania State Employees Credit Union",
+      "locationSet": {"include": ["us"]},
```

```suggestion
      "locationSet": {"include": ["us-pa.geojson"]},
```
I'm assuming PSECU only has branches in Pennsylvania